src/components/templates/collections-table/use-collection-actions.tsx

```diff
@@ -0,0 +1,43 @@
+import { navigate } from "gatsby"
+import { useAdminDeleteCollection } from "medusa-react"
+import * as React from "react"
+import useImperativeDialog from "../../../hooks/use-imperative-dialog"
+import EditIcon from "../../fundamentals/icons/edit-icon"
+import TrashIcon from "../../fundamentals/icons/trash-icon"
+import { ActionType } from "../../molecules/actionables"
+
+const useCollectionActions = (collection) => {
+  const dialog = useImperativeDialog()
+  const deleteCollection = useAdminDeleteCollection(collection?.id)
+
+  const handleDelete = async () => {
+    const shouldDelete = await dialog({
```

Comment:
I can't seem to cancel the delete dialog? Same when trying to delete a product in the list view.

Reply:
Have created an issue for this one 👍